src/mario_dream/castroll_data.asm

```diff
@@ -0,0 +1,914 @@
+	.db $F9, $E0, $00, $30 ; $28
+	.db $F9, $EA, $00, $38 ; $2C
+	.db $F9, $D4, $00, $40 ; $30
+	.db $F9, $D8, $00, $48 ; $34
+	.db $F9, $F4, $00, $50 ; $38
+	.db $F9, $F4, $00, $58 ; $3C
+CastRoll_Toad:
+	.db $D0, $3E, $00, $30 ; DATA XREF: BANKC:8B0Bo
+	.db $D0, $18, $00, $38 ; 4
+	.db $D0, $1A, $00, $40 ; 8
+	.db $D0, $3E, $00, $48 ; $C
+	.db $F9, $3E, $00, $30 ; $10
+	.db $F9, $1C, $00, $38 ; $14
+	.db $F9, $1E, $00, $40 ; $18
+	.db $F9, $3E, $00, $48 ; $1C
+	.db $F9, $3E, $00, $20 ; $20
+	.db $F9, $3E, $00, $28 ; $24
+	.db $F9, $F6, $00, $30 ; $28
+	.db $F9, $EC, $00, $38 ; $2C
+	.db $F9, $D0, $00, $40 ; $30
+	.db $F9, $D6, $00, $48 ; $34
+	.db $F9, $3E, $00, $50 ; $38
+	.db $F9, $3E, $00, $58 ; $3C
+CastRoll_Shyguy:
+	.db $D0, $3E, 0, $30
+	.db $D0, $3E, $00, $38 ; 4
+	.db $D0, $3E, $00, $40 ; 8
+	.db $D0, $3E, $00, $48 ; $C
+	.db $F9, $3E, $00, $30 ; $10
+	.db $F9, $20, $00, $38 ; $14
+	.db $F9, $22, $00, $40 ; $18
+	.db $F9, $3E, $00, $48 ; $1C
+	.db $F9, $3E, $00, $20 ; $20
+	.db $F9, $F4, $00, $28 ; $24
+	.db $F9, $DE, $00, $30 ; $28
+	.db $F9, $CC, $00, $38 ; $2C
+	.db $F9, $DC, $00, $40 ; $30
+	.db $F9, $F8, $00, $48 ; $34
+	.db $F9, $CC, $00, $50 ; $38
+	.db $F9, $3E, $00, $58 ; $3C
+CastRoll_Snifit:
+	.db $D0, $3E, 0, $30
+	.db $D0, $3E, $00, $38 ; 4
+	.db $D0, $3E, $00, $40 ; 8
+	.db $D0, $3E, $00, $48 ; $C
+	.db $F9, $3E, $00, $30 ; $10
+	.db $F9, $24, $00, $38 ; $14
+	.db $F9, $26, $00, $40 ; $18
+	.db $F9, $3E, $00, $48 ; $1C
+	.db $F9, $3E, $00, $20 ; $20
+	.db $F9, $F4, $00, $28 ; $24
+	.db $F9, $EA, $00, $30 ; $28
+	.db $F9, $E0, $00, $38 ; $2C
+	.db $F9, $DA, $00, $40 ; $30
+	.db $F9, $E0, $00, $48 ; $34
+	.db $F9, $F6, $00, $50 ; $38
+	.db $F9, $3E, $00, $58 ; $3C
+CastRoll_Ninji:
+	.db $D0, $3E, $00, $30
+	.db $D0, $3E, $00, $38 ; 4
+	.db $D0, $3E, $00, $40 ; 8
+	.db $D0, $3E, $00, $48 ; $C
+	.db $F9, $3E, $00, $30 ; $10
+	.db $F9, $28, $00, $38 ; $14
+	.db $F9, $2A, $00, $40 ; $18
+	.db $F9, $3E, $00, $48 ; $1C
+	.db $F9, $3E, $00, $24 ; $20
+	.db $F9, $EA, $00, $2C ; $24
+	.db $F9, $E0, $00, $34 ; $28
+	.db $F9, $EA, $00, $3C ; $2C
+	.db $F9, $E2, $00, $44 ; $30
+	.db $F9, $E0, $00, $4C ; $34
+	.db $F9, $3E, $00, $54 ; $38
+	.db $F9, $3E, $00, $5C ; $3C
+CastRoll_Beezo:
+	.db $D0, $3E, $00, $30
+	.db $D0, $3E, $00, $38 ; 4
+	.db $D0, $3E, $00, $40 ; 8
+	.db $D0, $3E, $00, $48 ; $C
+	.db $F9, $3E, $00, $30 ; $10
+	.db $F9, $2C, $00, $38 ; $14
+	.db $F9, $2E, $00, $40 ; $18
+	.db $F9, $3E, $00, $48 ; $1C
+	.db $F9, $3E, $00, $24 ; $20
+	.db $F9, $D2, $00, $2C ; $24
+	.db $F9, $D8, $00, $34 ; $28
+	.db $F9, $D8, $00, $3C ; $2C
+	.db $F9, $CE, $00, $44 ; $30
+	.db $F9, $EC, $00, $4C ; $34
+	.db $F9, $3E, $00, $54 ; $38
+	.db $F9, $3E, $00, $5C ; $3C
+CastRoll_Porcupo:
+	.db $D0, $3E, $00, $30
+	.db $D0, $3E, $00, $38 ; 4
+	.db $D0, $3E, $00, $40 ; 8
+	.db $D0, $3E, $00, $48 ; $C
+	.db $F9, $3E, $00, $30 ; $10
+	.db $F9, $30, $00, $38 ; $14
+	.db $F9, $32, $00, $40 ; $18
+	.db $F9 ; $1C
+byte_BANKC_8D5F:
```

Comment:
this looks like an extraneous auto label. i think you can remove this and reformat the lines up until `CastRoll_Tweeter` into groups of 4 bytes